metrics/ci_integrations/test/cli/command/sync_command_test.dart

```diff
@@ -273,20 +271,20 @@ void main() {
       );
 
       test(
-        ".sync() prints an error message if a sync result is an error",
+        ".sync() throws a SyncError if a sync result is an error",
```

Comment:
```suggestion
        ".sync() throws a sync error if a sync result is an error",
```
Or keep as is but change in the similar test cases in the above.